js/bootstrap-datepicker.js

```diff
@@ -1820,11 +1821,11 @@
 					fn = fn_map[part[2].toLowerCase()];
 					date = Datepicker.prototype[fn](date, dir);
 				}
+				// Use Datepicker.prototype._zero_utc_time in future
```

Comment:
Is this just some extra info or something we should do in the future? If so add `TODO:` before the comment so we can find those todo's easily